src/rules/noInferredEmptyObjectTypeRule.ts

```diff
@@ -30,7 +30,7 @@ export class Rule extends Lint.Rules.TypedRule {
         options: null,
         optionExamples: [true],
         rationale: Lint.Utils.dedent`
-            Prior to TypeScript 3.4, generic type parameters for functions and constructors are inferred as
+            Prior to TypeScript 3.5, generic type parameters for functions and constructors are inferred as
```

Comment:
> This PR also fixes a bit of documentation in the no-inferred-empty-object rule (it previously said "prior to 3.4", but it really is supposed to be "prior to 3.5". I didn't want to create another PR just for that, but can if that is y'all's call).

Yes please! Separate PRs are preferable to make it easier to work on and review things in parallel, especially so that requested changes to one area don't block accepting others.

Reply:
Got it! Will revert this.